test/dynamo/test_compile.py

```diff
@@ -110,6 +112,15 @@ def g(x):
             return x + x
 
         g(torch.randn(3))
+    
+    @unittest.skipIf(not TEST_CUDA, "requires CUDA")
```

Comment:
Nit: we have a generic mechanism for creating a device specific tests and decorators like `onlyCPU` or `onlyCUDA`. Why not use those here? (I.e. this change should probably work for ROCm as well)